### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by hamsajaisal** [aswinshenoy/bayjdo#6]
I am a visually impaired and using browser with the help of screen reader. I tried to use this service but few buttons were and labelled and I couldn't get code to send the other user. The two  buttons while we opening the website are spoken with the same name by the screen reader. Kindly lable the buttons and make almost all the fields accessible with the screen reader. Then we visually impaired people also can use the service.

**Comment by aswinshenoy** [comment]
I am extremely sorry 🙏 to know that you faced such an issue. 
I have shown an unacceptable careless attitude towards Web Accessibility, and I sincerely apologize to you for the same. 
I would truly want the platform to be accessible to everyone❤️,  and for this **I have put this issue on priority No.1 for me**  and will push an update as early as tomorrow!
Thank you so much! I look forward to hearing from you experience, once I make this accessible :)

**Comment by aswinshenoy** [comment]
I have worked on adding aria labels to most buttons and input fields I found, however, I have not yet tested it with a screen reader so don't know how far efficient it would be.
Also, I can very well understand your problem that you are unable to copy the peer code since screen-reader fails to pick it up especially/differentiate it among other things on the page, and also I found the screen reader instead pronounces the words forming the code instead of spelling it which also could be a issue. I am yet to fix those issues, and I will try to do within a few days, since I need to learn some more accessibility practices for doing so.

Thank You So Much once again 
I am totally humbled by the effort you have taken to put up an issue citing this,  and it was really a eye opener to me :)

**Comment by aswinshenoy** [comment]
I didn't get much time the last two days for making the app 💯% accessible (though most buttons and input fields are now labelled), since some critical bugs and optimizations work were pending. I shall continue to work on this issue parallel, and keep updates posted on this thread.

**Comment by hamsajaisal** [comment]
Hello there. Really happy to say that mani accessibility issues related to input fields and buttons are fixed. Thank you so much for the instant works made on it. By the way there are few buttons yet to be lebld properly. The two buttons for initiating file transfer are saying with the same name start transferring button. Though I didn't understand what is the difference between the two. Itne bhi 1 for sending and other one for receiving. If you could label the two buttons with send and receive or with its appropriates names that will be very much appreciated.

**Comment by hamsajaisal** [comment]
After clicking on start transfer button, the go back button is unlabelled.

**Comment by hamsajaisal** [comment]
Regarding copying the code, why can't we add a copy code button exclusively for screen reader users?

**Comment by hamsajaisal** [comment]
Dear brother. I am really new to GitHub.  I have no idea how to properly reply to a comment so you may lose connection while reading. Really sorry from my end. May I know can I contact you through any other platforms?? I have signal telegram and riot. Fortunately no WhatsApp and Facebook ☺️☺️ the problem is that, in GitHub, I am knowing that you have commented something after a lot of time. So hopefully you will reply.

**Comment by aswinshenoy** [comment]
> Itne bhi 1 for sending and other one for receiving

There is no seperate buttons for sending and receiving, the only option available is to connect with the peer and thereafter you shall be able to transfer both ways.


> The two buttons for initiating file transfer are saying with the same name start transferring button. Though I didn't understand what is the difference between the two.

Yeah you are right there are two buttons on intention that do the same thing, and labelled as "start transfer". Actually, one of those button is in the header section whereas the other is kept fixed at the bottom so that it remains visible even when one scrolls through the landing page. Both buttons do the same thing, and you can use any one of them.



> start transfer button, the go back button is unlabelled.

Sorry that I missed it, yeah actually on mobile devices there is a visible cross button that takes you back to the landing page, but I guess I left without labels. I will add labels to it asap.

> why can't we add a copy code button
Yes, certainly I guess a copy button for the code would do great even for non-screen reader users, and I shall try to integrate it asap.



> properly reply to a comment so you may lose connection

No issue brother/sir, I really appreciate your effort to come forward and point these out. You could actually use the "Quote Reply" option to reply comment on specific parts of my comment. (This is what I have done in this comment). Also, if you want some further assistance or talk in more depth regarding the issues you are facing or some other suggestions - reach me on telegram - @aswin_shenoy is my telegram username. You can also email me at aswinshenoy65@gmail.com.

Thank you once again, I assure you that I am commited to resolve the accessibility issues soon, and work on suggestions you pointed out.

**Comment by aswinshenoy** [comment]
> start transfer button, the go back button is unlabelled.

I have added labels to the back button which is visible when the "start transfer" window popups up and clicking on it will get you back to the landing page itself closing the popup.

> why can't we add a copy code button

I have made the area that displays the join code into a click to copy button, and added accessibility labels to it. so just by clicking on the button, the code is then copied to your device clipboard which can be then shared elsewhere.
